### PR TITLE
[Python] Extract Concepts from v2 exercise: reverse-string 

### PR DESCRIPTION
**Pull request by khoivan88**
Issue #656

### CODE REVIEW

File: languages/python/reference/exercise-concepts/reverse-string.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## Concept extraction for reverse-string
+
+Example implementation:
+```python
+def reverse(text: str) -> str:
+    """Reverse a given string
```
Comment (yawpitch) [inline languages/python/reference/exercise-concepts/reverse-string.md]: I think we can dispense with the docstring, it doesn't add anything but noise to the solution.

File: languages/python/reference/exercise-concepts/reverse-string.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+because `str` in python is a `sequence` object, slicing can be used here. Specifically: for syntax `string[start:stop:stride]`:
+  - `start`: 0-index of the start position, `start=0` by default (i.e., not specified) (start from the beginning)
+  - `stop`: 0-index of the stop position, `stop=-1` by default (i.e., not specified) (stop at the end)
+  - `stride`: number of skip step. For example, 
+     ```python
+     >>> string = 'ABCDEF'[::2]
+     >>> print(string)
+     'ACE'
+     ```
+     In this exercise, `stride = -1` means start from the end
```
Comment (yawpitch) [inline languages/python/reference/exercise-concepts/reverse-string.md]: This line is indented too far.

**Comment by yawpitch** [comment]
This is in a sufficient state that I'm going to suggest we merge it into the `python-concept-references` branch for cleanup and harmonization with the other "extract" PRs.

File: languages/python/reference/exercise-concepts/reverse-string.md (hunk 1 above)
Comment (khoivan88) [reply languages/python/reference/exercise-concepts/reverse-string.md]: Thank you for your suggestion.

File: languages/python/reference/exercise-concepts/reverse-string.md (hunk 2 above)
Comment (khoivan88) [reply languages/python/reference/exercise-concepts/reverse-string.md]: I see. Thank you for checking that. I thought I checked the Preview. I should learn markdown better!

**Pull request merged**